src/main/java/online/umbcraft/libraries/dupes/NetherPortalDupe.java

```diff
@@ -14,6 +15,7 @@
 import org.bukkit.inventory.InventoryHolder;
 import org.bukkit.inventory.ItemStack;
 
+import javax.swing.*;
```

Comment:
where are we using javax.swing?

Reply:
oh sorry Intellij made some imports and i forgot to remove them. All unused imports should be removed now